package.json

```diff
@@ -4,14 +4,15 @@
   "description": "The JavaScript implementation of the Dash Platform Protocol",
   "scripts": {
     "lint": "eslint .",
-    "test": "npm run test:coverage && npm run test:browsers",
+    "test": "(EXIT_CODE=0; npm run test:coverage || EXIT_CODE=$?; npm run test:browsers || EXIT_CODE=$?; npm run test:functional || EXIT_CODE=$?; exit $EXIT_CODE)",
```

Comment:
why do you need `EXIT_CODE=0;` here? looks hacky

Reply:
We want to report failure in the end so CI build fails, yet run all possible scripts/tests.